test/mustachio/render_tests_test.dart

```diff
@@ -18,6 +16,10 @@ void main() {
     var dartdocLibUri = await Isolate.resolvePackageUri(
         Uri.parse('package:dartdoc/dartdoc.dart'));
     var dartdocPath = p.dirname(p.dirname(dartdocLibUri.path));
+    // Correct Windows issue path coming out of [Isolate.resolvePackageUri].
+    if (dartdocPath.startsWith(p.windows.separator)) {
+      dartdocPath = dartdocPath.substring(1).replaceAll('/', p.separator);
```

Comment:
so weird that `pkg:path` doesn't "do the right thing" here!

Reply:
Yeah it is curious. I think the issue is that Isolate.resolvePackageUri gives a Uri like `file:///C:/one/two`, which maybe isn't incorrect because URIs should always have `/`, even on Windows? And then I ask for the `.path` out of that, which gives `/C:/one/two` and now we're in trouble.